python/tests/kat/t_extauth.py

```diff
@@ -939,19 +939,19 @@ def check(self):
 class AuthenticationGRPCVerTest(AmbassadorTest):
 
     target: ServiceType
-    specified_protocol_version: Literal['v2', 'v3', 'default']
-    expected_protocol_version: Literal['v2', 'v3']
+    specified_protocol_version: Literal['v3', 'default']
+    expected_protocol_version: Literal['v3']
     auth: ServiceType
 
     @classmethod
     def variants(cls) -> Generator[Node, None, None]:
-        for protocol_version in ['v2', 'v3', 'default']:
+        for protocol_version in ['v3', 'default']:
             yield cls(protocol_version, name="{self.specified_protocol_version}")
 
-    def init(self, protocol_version: Literal['v2', 'v3', 'default']):
+    def init(self, protocol_version: Literal['v3', 'default']):
         self.target = HTTP()
         self.specified_protocol_version = protocol_version
-        self.expected_protocol_version = "v2" if protocol_version == "default" else protocol_version
+        self.expected_protocol_version = "v3"
         self.auth = AGRPC(name="auth", protocol_version=self.expected_protocol_version)
 
     def config(self) -> Generator[Union[str, Tuple[Node, str]], None, None]:
```

Comment:
This test should
1. _not_ be dropping the `'v2'` case
2. checking that both the `v2` and `default` cases emit an error

Reply:
I see that there are unit tests for the IR emitting an error for `v2`, but I think this is something that is definitely worth having an e2e test for.

Reply:
I have added this back in and will see what CI says

Reply:
this feedback is no longer relevant due to pulling this commit out and already merging into master